examples/readme.md

```diff
@@ -51,6 +51,6 @@ Then run chromium and force it to use Quic.
 Now you can navigate to files in the `root` folder for example `https://localhost:4433/index.html`.
 
 ## Debugging
-The example [example client](https://github.com/hyperium/h3/examples/client.rs) can generate a `SSLKEYLOGFILE` to see the traffic unencrypted in tools like Wireshark.  
+The example [example client](../examples/client.rs) can generate a `SSLKEYLOGFILE` to see the traffic unencrypted in tools like Wireshark.  
```

Comment:
Nit: could be just 
```suggestion
The example [example client](client.rs) can generate a `SSLKEYLOGFILE` to see the traffic unencrypted in tools like Wireshark.  
```

Reply:
@eagr Thank you for your review :) & I like this suggestion.

I have committed this at ad3297e.